### PR TITLE
Add FP64 precision support in intel_npu

### PR DESCRIPTION
**Pull request by CailiWuIntel**
### Details:
Add FP64 precision pipeline support in intel_npu。

### Tickets:
 - E-121878

### CODE REVIEW

**Comment by CailiWuIntel** [comment]
RUN_CI_ALL

**Comment by ArtemySkrebkov** [comment]
There are no triggers like RUN_CI_ALL for OpenVINO repo. 

Also can you please update the description to avoid using a direct link? We normally post just number like this `E-121878`

**Comment by CailiWuIntel** [comment]
> There are no triggers like RUN_CI_ALL for OpenVINO repo.
> 
> Also can you please update the description to avoid using a direct link? We normally post just number like this `E-121878`

Thanks for your kindly reminder, the related ticket description is updated.

**Comment by PatrikStepan** [comment]
Can you please enable the relevant functional tests (for infer_request) as well in vpux-plugin repository?

**Review by ArtemySkrebkov — APPROVED** [review]
I am fine to merge like this and then follow up with a test in vpux repo.

File: src/plugins/intel_npu/src/backend/src/zero_infer_request.cpp [hunk 1]
```diff
@@ -277,9 +277,11 @@ void ZeroInferRequest::check_network_precision(const ov::element::Type_t precisi
         break;
     case ov::element::Type_t::i64:
         break;
+    case ov::element::Type_t::f64:
+        break;
     default:
         OPENVINO_THROW("Unsupported tensor precision: " + ov::element::Type(precision).get_type_name() +
-                       "! Supported precisions: FP32, FP16, U8, I8, U16, I16, U32, I32, U64, I64");
+                       "! Supported precisions: FP32, FP16, U8, I8, U16, I16, U32, I32, U64, I64, F64");
```
Comment (StefaniaHergane) [inline src/plugins/intel_npu/src/backend/src/zero_infer_request.cpp:284]:
```suggestion
                       "! Supported precisions: FP32, FP16, U8, I8, U16, I16, U32, I32, U64, I64, FP64");
```

Comment (CailiWuIntel) [reply src/plugins/intel_npu/src/backend/src/zero_infer_request.cpp]: Done at 04aecce.

**Review by StefaniaHergane — APPROVED** [review]

**Comment by CailiWuIntel** [comment]
Hi @ArtemySkrebkov  @StefaniaHergane, could you help to start CI, or tell how can I start it?
Also, could you help to merge this PR after CI pass, or tell me what else should I do?

**Comment by ArtemySkrebkov** [comment]
Started the CI checks.

**Comment by iefode** [comment]
Have started OV Jenkins CI pipeline

**Comment by PatrikStepan** [comment]
The relevant NPU functional tests are now part of OpenVINO repository as well: https://github.com/openvinotoolkit/openvino/tree/master/src/plugins/intel_npu/tests/functional/shared_tests_instances/behavior/ov_infer_request
Can we enable BehaviorTests/OVInferRequestCheckTensorPrecision*type=f64 now?

**Comment by CailiWuIntel** [comment]
> The relevant NPU functional tests are now part of OpenVINO repository as well: https://github.com/openvinotoolkit/openvino/tree/master/src/plugins/intel_npu/tests/functional/shared_tests_instances/behavior/ov_infer_request Can we enable BehaviorTests/OVInferRequestCheckTensorPrecision*type=f64 now?

I will try to delete the skip comment and run the test to check whether we can enable _BehaviorTests/OVInferRequestCheckTensorPrecision*type=f64_ or not.

**Comment by CailiWuIntel** [comment]
Hi, @iefode @PatrikStepan, could you help to check and start the CI.

**Comment by CailiWuIntel** [comment]
Hi @ArtemySkrebkov, could you help to start CI.

**Comment by ArtemySkrebkov** [comment]
Sure, started the CI checks.

**Comment by ArtemySkrebkov** [comment]
build_jenkins

**Comment by CailiWuIntel** [comment]
There are some changes in the current test plan, so this PR is not needed for now. I'll close it, thanks for your comments!

**Pull request merged**